packages/app/app/stacks/components/StacksTable.tsx

```diff
@@ -0,0 +1,191 @@
+
+const calculateAveragePrice = (order: Order) => {
+  let totalExecutedBuyAmount = 0;
+  let totalExecutedSellAmount = 0;
+  getCowOrders(order).forEach((cowOrder) => {
+    if (cowOrder.executedBuyAmount === "0") return;
+
+    totalExecutedBuyAmount += convertedAmount(
+      cowOrder.executedBuyAmount,
+      order.buyToken.decimals
+    );
+    totalExecutedSellAmount += convertedAmount(
+      cowOrder.executedSellAmount,
+      order.sellToken.decimals
+    );
+  });
+  const averagePrice = totalExecutedSellAmount / totalExecutedBuyAmount;
+  return averagePrice;
+};
+
+export const ordersDone = (order: Order) => {
+  return order.orderSlots.reduce((count, orderTimestamp) => {
+    if (Number(orderTimestamp) < currentTimestampInSeconds) return ++count;
+
+    return count;
+  }, 0);
+};
+
+const totalFundsUsed = (order: Order) =>
+  Number(buyAmountPerSlot(order)) * ordersDone(order);
+
+const buyAmountPerSlot = (order: Order) =>
+  convertedAmount(
+    Number(order.amount) / order.orderSlots.length,
+    order.buyToken.decimals
+  );
+
+const totalStacked = (order: Order) =>
+  getCowOrders(order).reduce((acc, cowOrder) => {
+    return (
+      acc + convertedAmount(cowOrder.executedBuyAmount, order.buyToken.decimals)
+    );
+  }, 0);
+
+const getPairSymbols = (order: Order) =>
+  `${order.buyToken.symbol}/${order.sellToken.symbol}`;
+
+export const StacksTable = ({ orders }: { orders: Order[] }) => (
+  <div className="w-full bg-white border rounded-3xl border-surface-50">
+    <Table>
+      <TableHeader>
+        <TableRow>
+          <TableHead>Stack</TableHead>
+          <TableHead className="text-right">Used funds</TableHead>
+          <TableHead className="text-right">Avg. Buy Price</TableHead>
+          <TableHead className="text-right">Progress</TableHead>
+          <TableHead className="text-right"></TableHead>
+        </TableRow>
+      </TableHeader>
+      <TableBody>
+        {orders.map((order) => (
+          <TableRow key={order.id}>
+            <TableCell className="flex items-center font-medium">
+              <div className="flex items-end">
+                <div className="flex items-center justify-center w-10 h-10 text-[10px] border-2 rounded-full border-primary-400 bg-primary-100">
+                  {order.buyToken.symbol.substring(0, 4)}
+                </div>
+                <div className="flex items-center justify-center w-5 h-5 -ml-2 text-[8px]  border-primary-400 border rounded-full bg-primary-200">
+                  {order.sellToken.symbol.substring(2, 5)}
+                </div>
+              </div>
+              <div className="ml-3 space-y-0.5">
+                <BodyText weight="bold">
+                  {totalStacked(order).toFixed(3)}
+                </BodyText>
+                <CaptionText className="text-em-low">
+                  {getPairSymbols(order)}
+                </CaptionText>
+              </div>
+            </TableCell>
+            <TableCell className="text-right">
```

Comment:
I thought I had comment but I didn't. Should we had `CellWrapper` styles into `TableCell`? I feel that having an extra component to add style isn't a good tradeoff. Or at least we could create a TableCell component with styles, no need for an extra div IMO. Also, if this component is only used here we can declare it in this file :)

Reply:
There are some `TableCell`'s that we do not want the same styles for eg:

```js
<TableCell className="flex items-center font-medium">
  <StackedTokenLogoPair order={order} />
    <div className="ml-3 space-y-0.5">
      <BodyText weight="bold">{totalStacked(order).toFixed(3)}</BodyText>
      <CaptionText className="text-em-low">
         {getOrderPairSymbols(order)}
       </CaptionText>
    </div>
  </TableCell>
```

Reply:
Also adding flex to a table `td` element breaks the table layout, and we do not want that.

<img width="1140" alt="image" src="https://github.com/SwaprHQ/stackly-ui/assets/5664434/ca819db8-a3d1-435c-8177-8d5db63019a9">

Reply:
Interesting, I would take this, but the problem here is that it doesn't handle the mobile styles properly.

<img width="461" alt="image" src="https://github.com/SwaprHQ/stackly-ui/assets/5664434/f2d2b2bc-6a23-4463-a582-b08d01f4db70">